.github/workflows/test.yml

```diff
@@ -142,7 +142,7 @@ jobs:
         with:
           # Please adjust README and rust-version field in Cargo.toml files when
           # bumping version.
-          toolchain: 1.78.0
+          toolchain: 1.82.0
```

Comment:
Can you please create a separate commit for the MSRV bump? Let's please also bump it in `libbpf-rs`'s README and add a CHANGELOG entry there as well (we haven't done that consistently but it makes the most sense; we don't test both in isolation).

Reply:
Created a separate commit, let me know if you'd rather a separate PR too.